Projects/App/Sources/Journeys/MainJourney.swift

```diff
@@ -13,6 +13,10 @@ extension AppJourney {
                 UpdateAppScreen.journey.onPresent {
                     Launch.shared.completeAnimationCallbacker.callAll()
                 }
+            } else if Dependencies.featureFlags().osVersionTooLow {
```

Comment:
Can we do this like this?
```
 if Dependencies.featureFlags().isUpdateNecessary {
                if Dependencies.featureFlags().osVersionTooLow {
                    UpdateOSScreen.journey.onPresent {
                        Launch.shared.completeAnimationCallbacker.callAll()
                    }
                } else {
                    UpdateAppScreen.journey.onPresent {
                        Launch.shared.completeAnimationCallbacker.callAll()
                    }
                }
            } else {
               ....
            }
```

Why? We only want to show this when we force member to update the app.